src/docfx/lib/git/CommitBuildTimeProvider.cs

```diff
@@ -31,7 +32,12 @@ public CommitBuildTimeProvider(Repository repo)
 
         public bool TryGetCommitBuildTime(string commitId, out DateTime time)
         {
-            return _buildTimeByCommit.TryGetValue(commitId, out time);
+            if (!_buildTimeByCommit.TryGetValue(commitId, out time))
+            {
+                time = _buildTime;
```

Comment:
why setting `time` here, seems it will only be used when `commitId` found in `_buildTimeByCommit`:
https://github.com/dotnet/docfx/blob/56c5fa22bc9122e039c78f05d7fa58d302cfeec5/src/docfx/build/metadata/ContributionProvider.cs#L147

Reply:
it should always return false